.github/workflows/ci.yml

```diff
@@ -26,6 +26,20 @@ jobs:
            env:
              RAILS_VERSION: 'main'
 
+         # Rails 7.0 builds >= 2.7
+         - ruby: 3.1
+           allow_failure: true
+           env:
+             RAILS_VERSION: '~> 7.0.0'
+         - ruby: 3.0
```

Comment:
Hi 🙌  

You should use '3.0' otherwise github actions will pull 3.1.  Related to this https://github.com/actions/runner/issues/849 / https://github.com/ruby/setup-ruby/issues/252
![image](https://user-images.githubusercontent.com/16997/147757436-ed0c5262-4b88-4571-9b9b-53deaec23935.png)

I guess, all other ruby versions should be quoted too? 🤔

Reply:
Thanks for noticing!
I guess only `x.0` versions are affected.